pdns/lua-recursor4.cc

```diff
@@ -343,6 +343,12 @@ RecursorLua4::RecursorLua4(const std::string& fname)
   d_lw->registerFunction("getRecords", &DNSQuestion::getRecords);
   d_lw->registerFunction("setRecords", &DNSQuestion::setRecords);
 
+  d_lw->writeFunction("addPolicyTag", [](vector<string>* tags, const string& tag) {
+      if (tags) {
+        tags->push_back(tag);
+      }
+    });
+
```

Comment:
I always forget that LuaWrapper handles returning multiple values. I'll try to make option 2 work, I like it a lot better than the current one.